P0009/P0009.md

```diff
@@ -2305,13 +2304,15 @@ template<class StridedLayoutMapping>
 
    * [4.4]{.pnum} `StridedLayoutMapping::is_always_strided()` is `true`.
 
-[5]{.pnum} *Precondition:* 
+[5]{.pnum} *Preconditions:*
```

Comment:
Also add precondition from above constructor about "there exists a permutation...," since that precondition may not apply to arbitrary `StridedLayoutMapping`.

Reply:
Better yet, impose precondition that offset is zero, then effects equivalent to the above constructor.

Reply:
Done locally; will push.